src/Image_Generators/DNA_Generator.py

```diff
@@ -360,11 +363,12 @@ def send_To_Record_JSON():
 # THIS WILL DELETE THE RECORD:
 # Note - NFTRecrod.json will be created the next time you run main.py
 def clearNFTRecord(AREYOUSURE):
-   if AREYOUSURE == True:
+   if AREYOUSURE is True:
```

Comment:
@FastestMolasses for checking if a bool is True or False I am now using the `if variable:` and `if not variable:` method. Could you please change this?